tests/test_transformDiaSourceCatalog.py

```diff
@@ -175,6 +175,7 @@ def test_computeBBoxSize(self):
     def test_bit_unpacker(self):
         """Test that the integer bit packer is functioning correctly.
         """
+        self.config.doPackFlags = True
```

Comment:
Didn't  you deprecate this above?

Reply:
Yes, but I'm keeping the tests for the now non-default functionality.